install.yml

```diff
@@ -11,7 +11,7 @@ requirements:
     channels:
       - conda-forge
     packages:
-      - django=3.2.*
+      - django=5.2.*
```

Comment:
I'd prefer this to be `>=3.2,<6` for flexibility? Will that work with CityWater?